packages/jsapi-components/package.json

```diff
@@ -22,6 +22,7 @@
     "build:sass": "sass --embed-sources --load-path=../../node_modules ./src:./dist"
   },
   "dependencies": {
+    "@adobe/react-spectrum": "^3.34.1",
```

Comment:
Should import everything from `@deephaven/components`, even in `jsapi-components`. Seems we're importing `useProvider` in `ListView.tsx`.

Reply:
@mofojed  hmm, we aren't re-exporting currently. Any opinions on how we want to do this? `useProvider` seems too generic, but `useSpectrumProvider` seems a bit odd since we are trying to abstract away Spectrum.

Reply:
Well, you're exporting `SpectrumThemeProvider`, `useSpectrumThemeProvider` to mirror that would be fine with me.
Unsure if we want to get more abstract than that, we already have `ThemeProvider`.